driver-core/src/test/java/com/datastax/driver/core/AuthenticationTest.java

```diff
@@ -76,12 +76,7 @@ public void should_fail_to_connect_with_wrong_credentials() throws InterruptedEx
                 .withPort(ccm().getBinaryPort())
                 .withCredentials("bogus", "bogus")
                 .build());
-    try {
-      cluster.connect();
-    } finally {
-      assertThat(cluster.getMetrics().getErrorMetrics().getAuthenticationErrors().getCount())
-          .isEqualTo(1);
-    }
+    cluster.connect();
   }
 
```

Comment:
We now close the cluster on initialization errors. This means the cluster will be closed, when we call getMetrics, so this check is no longer applicable.